pom.xml

```diff
@@ -77,6 +77,12 @@
   </pluginRepositories>
 	<dependencyManagement>
 		<dependencies>
+          <dependency>
```

Comment:
nit: Please make the indents similar to the rest of the file (i.e., tab vs space).